packages/hash/frontend/src/blocks/page/Comments/CommentTextField.tsx

```diff
@@ -0,0 +1,194 @@
+  );
+
+  const createEditor = useCallback(
+    (container: HTMLDivElement) => {
+      const schema = createSchema({
+        doc: {
+          content: "inline*",
+        },
+        text: textNode,
+        hardBreak: hardBreakNode,
+        mention: mentionNode,
+      });
+
+      const doc = schema.node("doc", {}, []);
+
+      const state = EditorState.create<Schema>({
+        doc,
+        plugins: [
+          keymap<Schema>(baseKeymap),
+          ...createFormatPlugins(renderPortal),
+          formatKeymap(doc),
+          createSuggester(renderPortal, accountId, container),
+          placeholderPlugin(renderPortal, "Leave a comment"),
+        ],
+      });
+
+      const view = new EditorView<Schema>(container, {
+        state,
+        clipboardTextSerializer: clipboardTextSerializer(
+          state.schema.nodes.hardBreak,
+        ),
+        nodeViews: {
+          mention: mentionNodeView(renderPortal, accountId),
+        },
+        handleKeyDown: (_, { shiftKey, key }) => {
+          if (!shiftKey && key === "Enter") {
+            if (onSubmit && viewRef.current?.state.doc.content) {
+              const tokens =
+                viewRef.current.state.doc.content.toJSON() as TextToken[];
+
+              setLoading(true);
+              onSubmit(tokens)
+                .then(() => {
+                  onClose();
+                })
+                .finally(() => {
+                  setLoading(false);
+                });
+            }
+
+            return true;
+          }
+
+          return false;
+        },
+      });
+
+      view.dom.classList.add(styles.Prosemirror_Input!);
+
+      view.focus();
+
+      viewRef.current = view;
+    },
+    [accountId, onSubmit, renderPortal, onClose],
+  );
+
+  if (editorContainer && prevBlockId !== blockId) {
+    setPrevBlockId(blockId);
+
+    editorContainer.innerHTML = "";
+    createEditor(editorContainer);
+  }
+
+  return (
+    <Box
+      sx={({ transitions, palette }) => ({
+        width: 250,
+        display: "flex",
+        borderRadius: 1.5,
+        border: `1px solid ${palette.gray[30]}`,
+        backdropFilter: "blur(40px)",
+        transition: transitions.create("border-color"),
+        "&:focus-within": {
+          borderColor: palette.blue[60],
+        },
+      })}
+    >
+      <IconButton
+        onClick={onClose}
+        sx={({ palette }) => ({
+          padding: 0.5,
+          borderRadius: 1,
+          margin: 1.5,
+          alignSelf: "flex-start",
+          color: palette.gray[50],
+        })}
+      >
+        <FontAwesomeIcon icon={faComment} />
+      </IconButton>
+
+      <Box
+        ref={(ref: HTMLDivElement) => setEditorContainer(ref)}
+        sx={({ palette }) => ({
+          overflow: "hidden",
+          flexGrow: 1,
+          fontSize: 14,
+          lineHeight: "150%",
+          color: palette.gray[90],
+        })}
+      />
+
+      <Box sx={{ display: "flex", alignItems: "flex-end", margin: 1.5 }}>
```

Comment:
```suggestion
      <Box display="flex" alignItems="flex-end" margin={1.5}>
```

Suggesting this based on the information at [MUI Best Practices - Use the MUI Box component convenience props consistently](https://www.notion.so/hashintel/MUI-Best-Practices-3031a2907df64274ac88aa57534d3e39#cbc0734185524e83989227933bc477cd) _(internal)_

Same suggestion could apply few more `Box`'es on this page. Not creating comments for all of them.

Reply:
Made this change in 00a73809338520d65dbfe9182fdcdf108186fc96.